content/toolsAndResources/nge.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Node Geometry Editor
+image: 
+description: Learn how use the Node Geometry Editor
+keywords: geometry, procedural, non destructive, node, graph
+further-reading:
+video-overview:
+video-content:
+---
+
+## What Is Node Geometry Editor?
+The Node Geometry Editor will be familiar for anyone who has already used the Node Material Editor. This editor, like the Node Material Editor, was created to speed up the process of creating Node Geometry and reduce the amount and complexity of code that needs to be written to generate Node Geometry. Additionally, users will not need deep knowledge of the Node Geometry Class as the tool handles all of the necessary syntax and validation for us. 
+
+The tool is available as a stand alone experience by visiting [Node Geometry Editor](https://nge.babylonjs.com) and can also be opened from the inspector when selecting any Node Geometry in scene.
+
+![Node Geometry Editor Launch Button](/img/tools/nge/ngeButton.jpg)
+
+## Interface
+
+The interface for the editor is broken into 4 main sections as can be seen below. The node list is the entire list of available nodes that can be dragged onto the graph. At the top of the list is a search box to help narrow the list. The graph is the workspace for arranging the nodes to create the geometry flow. There is a panel for parameters and controls on the right side of the window which displays any parameters available to the currently selected node. If nothing on the graph is selected, this panel shows controls for the tool itself to load, save, or change default settings. In the bottom right is the preview window which displays the current Node Geometry output. Lastly, at the bottom of the window is the console log panel which gives information the build status of the current graph. 
+
+![Babylon.js Node Geometry Editor](/img/tools/nge/ngeWindow.jpg)
+
+The image above shows three types of nodes available in the graph, an input node, a source node, and an output node. At minimum all graphs need exactly one output node and at least one source node. Note that the inputs and outputs on the nodes share colors and icons that give hints on what can be connected. Since the `Geometry Output` node has only a geometry input, there needs to be at least one node supplying that that input.

```

Comment:
nit: "that that" => that

Reply:
updated in https://github.com/BabylonJS/Documentation/pull/963/commits/07fc54158e3862125b1824623a58ae435c2dd890